services/web/server/src/simcore_service_webserver/exporter/_handlers.py

```diff
@@ -42,18 +46,21 @@ async def export_project(request: web.Request):
     user_id = request[RQT_USERID_KEY]
     project_uuid = request.match_info.get("project_id")
     assert project_uuid  # nosec
-    delete_tmp_dir: Callable[[], Coroutine[Any, Any, None]] | None = None
-    try:
-        async with AsyncExitStack() as tmp_dir_stack, lock_project(
-            request.app,
-            project_uuid,
-            ProjectStatus.EXPORTING,
-            user_id,
-            await get_user_fullname(request.app, user_id=user_id),
-        ):
-            await retrieve_and_notify_project_locked_state(
-                user_id, project_uuid, request.app
-            )
+
+    @with_project_locked(
+        get_redis_lock_manager_client_sdk(request.app),
+        project_uuid=project_uuid,
+        status=ProjectStatus.EXPORTING,
+        owner=Owner(
+            user_id=user_id, **await get_user_fullname(request.app, user_id=user_id)
+        ),
+        notification_cb=create_user_notification_cb(
+            user_id, ProjectID(f"{project_uuid}"), request.app
+        ),
+    )
+    async def _() -> tuple[Callable[[], Coroutine[Any, Any, None]], Path]:
+        # @GitHK what is this supposed to be doing??
```

Comment:
Q: what "this" are you referring to here? can you elaborate your question a bit more?

Reply:
yes. 2 things:
- exporter is off for years, can we get rid of that code?
- I am not sure I fully understand what kind of coroutine it is returning.
Anyway will remove the comment